.github/workflows/main.yml

```diff
@@ -0,0 +1,67 @@
+name: main
+on:
+  - push
+  - pull_request
+jobs:
+  rspec:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        ruby-version:
+          - 2.5
```

Comment:
FYI: I decided to include 2.5 (it's EOL now) for any case because looks like it's still using and because only this version is compatible with Rails 5.2 (it's still supporting).

Reply:
:+1: